core/primitives/src/block.rs

```diff
@@ -32,6 +32,8 @@ pub struct BlockHeaderInner {
     pub chunk_headers_root: MerkleHash,
     /// Root hash of the chunk transactions in the given block.
     pub chunk_tx_root: MerkleHash,
+    /// Number of chunks included into the block.
+    pub chunks_included: u64,
```

Comment:
You don't verify that `chunks_included` is correctly computed when receive a block